.github/actions/branch-build-and-push/action.yaml

```diff
@@ -53,7 +53,7 @@ runs:
           docker login ghcr.io -u qdrant --password "${{ inputs.ghcr-password }}"
         fi
         if [ "${{ inputs.push-to-dockerhub }}" = "true" ]; then
-          docker login docker.io -u qdrant --password "${{ inputs.dockerhub-password }}"
```

Comment:
Let's also fix the following warning in this PR:

```
WARNING! Using --password via the CLI is insecure. Use --password-stdin.
```

Reply:
Nit: if we are going full hardening, we'll should use `docker login ... <<< "${...}"` instead of `echo "${...}" | docker login ...`. This way, this value will be avoided in the logs:

```console
$ set -x
+ set -x

$ echo topsecret | md5sum -
+ md5sum -
+ echo topsecret
072692aa1a2d7c2eaece991897f5026d  -

$ md5sum - <<< topsecret
+ md5sum -
072692aa1a2d7c2eaece991897f5026d  -
```

(tho, realistically, these values will be filtered out from the logs by GitHub Actions)